google-spark-operator/operator/base/Kube-descriptor.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: v1beta2-1.2.3-3.1.1
+description: Spark operator based on https://github.com/GoogleCloudPlatform/spark-on-k8s-operator
+home: https://github.com/opendatahub-io/odh-manifests/google-spark-operator
+keywords:
+- spark
+kubeVersion: '>=1.8.0-0'
+name: sparkoperator
+version: 0.0.1
```

Comment:
Is this file required? At least it is not listed in kustomize.yaml

Reply:
File removed